pkg/machines/config.es6

```diff
@@ -25,7 +25,14 @@
 const VMS_CONFIG = {
     DefaultRefreshInterval: 10000, // in ms
     Virsh: {
-        ConnectionParams: ['-c', 'qemu:///system']
+        connections: {
+            'system': {
```

Comment:
Are these translatable connection names? Are they translated in virt-manager or elsewhere?